lazycoder1412/3SumSmaller.cpp

```diff
@@ -0,0 +1,18 @@
+class Solution {
+public:
+    int threeSumSmaller(vector<int>& nums, int target) {
+        sort(nums.begin(),nums.end());
```

Comment:
Commas should always be separated with a space as so - `sort(nums.begin(), nums.end());`